todo_app/templates/index.html

```diff
@@ -11,7 +11,11 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+
+        {% for item in items %}
+           <li>{{ item.title }} - {{ item.status }} - {{ item.main }}</li>
```

Comment:
What is the purpose of the `main` attribute on an item? It appears from above that it is just the status of the item. Can we remove the call to `item.main`?